app/services/authentication.py

```diff
@@ -22,10 +26,14 @@ def generate_registration_url(token):
 
 
 def send_registration_email(email_address):
-    token = encode_token(email_address)
+    token = encode_token(email_address, "email-confirm-salt")
```

Comment:
you should put the salt into vault and retrieve it from there or just salt it with the app's secret key.

Reply:
https://pythonhosted.org/itsdangerous/#the-salt

Reply:
I see.   Well - that's a weird thing to call a salt I guess.   Okay - it doesn't matter if these salts are public, because they're not really "salts" in the crypto sense.

Reply:
Yes horrible naming